README.md

```diff
@@ -1,2 +1,66 @@
-# reggaemedia
-the reggaemedia website
+# Getting started
+1. Before starting development server you need to define environment variables.
+2. To be able to deploy project stack, your PATH should include executable aws package with configured IAM account.
+3. Preferred package manager is `yarn` version 2.
+
+## .ENV file
+Environment variables file must include vars written below to be able to deploy AWS stack. Vars below is stale and should be define once for whole project.
+
+```
+COGNITO_CALLBACK_URL=http://localhost:3000/api/auth/callback/cognito
+ADMIN_EMAIL
+CDK_DEFAULT_ACCOUNT= default account that is being defined in aws configuration
+CDK_DEFAULT_REGION= preferred region
+AMAZON_ACCESS_KEY= IAM credentials
+AMAZON_SECRET_KEY= IAM credentials
+AMAZON_REGION= preferred region
+```
+
+After defining variables above you should run command `yarn stack:deploy`. That command will generate `cdk-outputs.json` that is being gitignored, output file contain another variables that must be defined in the same .env file as above. 
+
+### WARNING:
+After running `yarn stack:deploy`, you always have to update your .env file, since stack is being regenerated.
+
+```
+{
+  "reggae-media-stack": {
+    "bucket",
+    "settings",
+    "userPoolClientId",
+    "userPoolClientSecret",
+    "articles",
+    "userPoolId",
+  }
+}
+```
+
+Environments below should contain data that is being generated by aws-cdk-lib in `cdk-outputs.json`.
+
+```
+BUCKET_NAME="bucket"
+TABLE_ARTICLES="articles"
+COGNITO_CLIENT_SECRET="userPoolClientSecret"
+COGNITO_CLIENT_ID="userPoolClientId"
+COGNITO_USER_POOL_ID="userPoolId"
+TABLE_SETTINGS="settings"
+```
+
+## Development process
+After deploying your stack and defining all the variables run command `yarn dev` to start development process.
+
+To be able to get full access to all protected and public paths you should create an user pool account, that MUST include Confirmation Status as Confirmed.
```

Comment:
This can be stated under FAQ or Useful information topic or etc

Also it's nice to have following commands also:

add user:
`
aws cognito-idp admin-create-user \
  --user-pool-id <PoolId> \
  --username <email>
`

set user name:
`
aws cognito-idp admin-update-user-attributes \
  --user-pool-id <PoolId> \
  --username <email> \
  --user-attributes Name="name",Value="<Full Name>"
`

confirm user:
`
aws cognito-idp admin-set-user-password \
  --user-pool-id <PoolId> \
  --username <email> \
  --password <password> \
  --permanent
`